package.json

```diff
@@ -70,14 +70,13 @@
     "jsdom": "15.1.1",
     "koa-connect": "2.0.1",
     "mini-css-extract-plugin": "0.8.0",
-    "node-sass": "4.14.1",
+    "node-sass": "^4.14.1",
     "npm-check-updates": "3.1.24",
     "optimize-css-assets-webpack-plugin": "5.0.3",
     "prettier": "2.2.1",
     "pretty-quick": "1.11.1",
     "react-hot-loader": "4.13.0",
     "redux-immutable-state-invariant": "2.1.0",
-    "sass-loader": "8.0.0",
```

Comment:
what are those changes needed for?

Reply:
I made some necessary configuration changes in order to work with node js version 16. Also I think that I will not use 'sass' for this app. Hope it is acceptable.